modules/cluster-logging-troubleshooting-loki.adoc

```diff
@@ -0,0 +1,108 @@
+.Troubleshooting
+
+Issue: Forwarding a large block of messages exceeds the rate limit and causes errors.
+
+Identification:
+* You have configured the `ClusterLogForwarder` custom resource to forward logs to Grafana Loki.
+* A large block of messages, larger than 2 MB, is sent to Loki, such as:
++
+----
+"values":[["1630410392689800468","{\"kind\":\"Event\",\"apiVersion\":\
+.......
+......
+......
+......
+\"received_at\":\"2021-08-31T11:46:32.800278+00:00\",\"version\":\"1.7.4 1.6.0\"}},\"@timestamp\":\"2021-08-31T11:46:32.799692+00:00\",\"viaq_index_name\":\"audit-write\",\"viaq_msg_id\":\"MzFjYjJkZjItNjY0MC00YWU4LWIwMTEtNGNmM2E5ZmViMGU4\",\"log_type\":\"audit\"}"]]}]}
+----
+* TBD displays the following messages:
++
+[source,text]
+----
+429 Too Many Requests Ingestion rate limit exceeded (limit: 8388608 bytes/sec) while attempting to ingest '2140' lines totaling '3285284' bytes
+
+429 Too Many Requests Ingestion rate limit exceeded' or '500 Internal Server Error rpc error: code = ResourceExhausted desc = grpc: received message larger than max (5277702 vs. 4194304)'
+----
+
+Solution: In TBD file, update the following fields to use the values shown:
```

Comment:
@anpingli What is the name/location of this file?

Reply:
Loki Config file